src/Etg.Yams.Core/YamsConfigBuilder.cs

```diff
@@ -21,6 +21,7 @@ public class YamsConfigBuilder
             _appHeartBeatTimeout = TimeSpan.FromSeconds(60);
             _ipcConnectTimeout = TimeSpan.FromSeconds(60);
             _appInitTimeout = TimeSpan.FromSeconds(60);
+            _updateSessionTtl = TimeSpan.FromHours(1);
         }
 
         public YamsConfigBuilder SetCheckForUpdatesPeriodInSeconds(int value)
```

Comment:
value passed in not used?

Reply:
TTL is more for leftover rows if an instance has crashed and left the cluster in an inconsistent state. This should be very rare. The advantage of TTL is that users won't have to manually clear the table in such cases.

Reply:
My comment made it to the wrong place, I was referring to the SetCheckForUpdatesPeriodInSeconds method, which doesn't use the int value passed in.

Reply:
CheckForUpdatesPeriodInSeconds define the frequency at which Yams pools the `DeploymentConfig.json` for changes. There is a default value of 10 seconds but it can be changed using `SetCheckForUpdatesPeriodInSeconds`.

Reply:
No issue here, the diff view hides a line of code I thought it was missing. :)